mix.exs

```diff
@@ -197,6 +200,20 @@ defmodule FunWithFlags.Mixfile do
     )
   end
 
+  # Run the tests with Ecto+SQLite3 as persistent store and Phoenix.PubSub as broker.
+  #
+  defp run_tests__ecto_pers_sqlite3__phoenix_pubsub(arg) do
+    Mix.shell.cmd(
+      "mix test --color --force --exclude redis_pubsub --exclude redis_persistence #{arg}",
+      env: [
+        {"CACHE_ENABLED", "true"},
+        {"PUBSUB_BROKER", "phoenix_pubsub"},
+        {"PERSISTENCE", "ecto"},
+        {"RDBMS", "sqlite3"},
+      ]
+    )
+  end
```

Comment:
To keep the functions in a consistent order, I'd move this below `run_tests__ecto_pers_mysql__phoenix_pubsub`.